modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -2,22 +2,37 @@
 //
 // * installing/installing_bare_metal/installing-bare-metal.adoc
 // * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
-// * installing_bare_metal/installing-bare-metal-network-customizations.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
 
 [id="installation-user-infra-machines-static-network_{context}"]
-= Configure advanced networking
+= Advanced {op-system} installation reference
 
-If you install {op-system-first} from an ISO image, you can add kernel arguments
-when you boot that image to configure the node's networking.
-The following table describes and illustrates how to use those kernel arguments.
+This section illustrates the networking configuration and other advanced options that allow you to modify the {op-system-first} bare metal install process. The following tables describe the kernel arguments and command-line options you can use with:
 
-.Configure advanced networking
+* The {op-system} live installer boot prompt by using routing and bonding.
+
+* The {op-system} live installer by using `core.inst` for ISO or PXE installs.
```

Comment:
Good catch on the typos, thanks--fixed.

I've pared down the intro paragraph so that it is more generalized:

> This section illustrates the networking configuration and other advanced options that allow you to modify the RHCOS bare metal install process. The following tables describe the kernel arguments and command-line options you can use with the RHCOS live installer and the `coreos-installer` command.